stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -115,15 +115,27 @@ internal class PaymentSheetViewModel internal constructor(
     }
 
     private fun onPaymentIntentResponse(paymentIntent: PaymentIntent) {
-        runCatching {
-            paymentIntentValidator.requireValid(paymentIntent)
-        }.fold(
-            onSuccess = {
-                _paymentIntent.value = paymentIntent
-                resetViewState(paymentIntent)
-            },
-            onFailure = ::onFatal
-        )
+        if (paymentIntent.isConfirmed) {
```

Comment:
Can we log that the payment is already confirmed at INFO or WARNING? This is a case that might confuse developers.